public_profiler/v3.5-DD/ogl/skills.js

```diff
@@ -719,22 +719,22 @@ var spellsHelpURL = {
   "halt undead":                             baseHelpURL + "spells/haltUndead.htm",
   "harm":                                    baseHelpURL + "spells/harm.htm",
   "haste":                                   baseHelpURL + "spells/haste.htm",
-  "heal":                                    baseHelpURL + "spells/heal.htm",
   "heal, mass":                              baseHelpURL + "spells/healMass.htm",
+  "heal":                                    baseHelpURL + "spells/heal.htm",
```

Comment:
(Surely heal needs to go after heal mount, else the latter never gets matched ... OK, handled in a later commit.)